### ISSUE TITLE
After upgrading to 2.0.0-alpha.11, service no longer exists

### ISSUE THREAD

**Issue by mrdarrengriffin** [hyperion-project/hyperion.ng#1350]
- [x] I confirm that this is an issue rather than a question.

## Bug report
I upgraded to the new alpha.10.1 yesterday and it transpired that there was an issue with the build. After I upgraded hyperion, the service never came online.

I then install Hyperion alpha.11 following the release earlier but the service is not starting.

#### Steps to reproduce
- Upgraded to alpha.11 via `sudo apt-get update && sudo apt-get install hyperion`

#### What is expected?
Running `sudo systemctl hyperion` and `sudo systemtl hyperion.service` should show the status of the Hyperion service

#### What is actually happening?
Running the above results in:

`Unit hyperion.service could not be found.`

Tried to reinstall hyperion by running `sudo apt-get install --reinstall hyperion`. Although it completes, no service exists still

#### System

Unable to obtain. Running Raspberry Pi 4. OS info below
![image](https://user-images.githubusercontent.com/6920899/136199033-1b63f1c7-8fd5-4d54-ae45-273ab9157ea5.png)

**Comment by mrdarrengriffin** [comment]
No, I'm using it headless.

**Comment by Paulchen-Panther** [comment]
Version 9 did not exist as an APT version, hence the question.

**Comment by mrdarrengriffin** [comment]
Ah I see. Then it wouldn't been installed manually. In which case, I wouldn't have removed it. 

If this is an edge case, I'm fine with reinstalling but happy to assist

**Comment by mrdarrengriffin** [comment]
> Regarding this problem, I would ask our LEDDevice pro @Lord-Grey for help. Or create a new issue. As a precaution, I will leave this issue open if you want.

Thanks alot. I'll leave it up to @Lord-Grey whether or not to open a new issue

**Comment by fir3drag0n** [comment]
Same Problem, Same system here

**Comment by N1c093** [comment]
I have the same problem.
I had version Alpha 10 installed manually before. 

I updated to version Alpha 11. Sadly the service also doesn't exists. I also running raspbian headless.
As suggested I uninstalled hyperion with this command `sudo apt remove hyperion`. Afterwards I checked, if the hyperion folder exists under `/usr/share/` (It was removed)

I reinstalled Alpha 11 with this command `sudo apt install hyperion`

Sadly the service isn't installed afterwards.

@mrdarrengriffin Does the reinstall fixed the problem for you?

**Comment by mrdarrengriffin** [comment]
> I have the same problem. I had version Alpha 10 installed manually before.
> 
> I updated to version Alpha 11. Sadly the service also doesn't exists. I also running raspbian headless. As suggested I uninstalled hyperion with this command `sudo apt remove hyperion`. Afterwards I checked, if the hyperion folder exists under `/usr/share/` (It was removed)
> 
> I reinstalled Alpha 11 with this command `sudo apt install hyperion`
> 
> Sadly the service isn't installed afterwards.
> 
> @mrdarrengriffin Does the reinstall fixed the problem for you?

It did, Try 
`
sudo apt-get remove hyperion
sudo apt-get purge hyperion
sudo apt-get update
sudo apt-get install hyperion
`

**Comment by Paulchen-Panther** [comment]
> I have the same problem. I had version Alpha 10 installed manually before.
> 
> I updated to version Alpha 11. Sadly the service also doesn't exists. I also running raspbian headless. As suggested I uninstalled hyperion with this command `sudo apt remove hyperion`. Afterwards I checked, if the hyperion folder exists under `/usr/share/` (It was removed)
> 
> I reinstalled Alpha 11 with this command `sudo apt install hyperion`
> 
> Sadly the service isn't installed afterwards.
> 
> @mrdarrengriffin Does the reinstall fixed the problem for you?

An installation log would be helpful, as well as the output of `export`

**Comment by Lord-Grey** [comment]
> Thanks alot. I'll leave it up to @Lord-Grey whether or not to open a new issue

@mrdarrengriffin If you do not mind, I would appreciate a separate issue that we have everything at one place.
Please provide more background on what device you are running, as well in which scenario this happens. If you could share a debug log after executing the scenario (just use copy to clipboard on the System/Log Screen) would be brilliant.

Many thanks in advance!

**Comment by N1c093** [comment]
Sadly the hint from @mrdarrengriffin doesn't helped.

Here is the install log.
It seems like, that the install think I'm using the raspberry with gui.

```
pi@hyperion:~ $ sudo apt-get install hyperion
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following NEW packages will be installed:
  hyperion
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 5,881 kB of archives.
After this operation, 26.9 MB of additional disk space will be used.
Get:1 https://apt.hyperion-project.org buster/main armhf hyperion armhf 2.0.0-alpha.11~buster [5,881 kB]
Fetched 5,881 kB in 2s (2,374 kB/s)
Selecting previously unselected package hyperion.
(Reading database ... 48908 files and directories currently installed.)
Preparing to unpack .../hyperion_2.0.0-alpha.11~buster_armhf.deb ...
---Hyperion ambient light preinst ---
Unpacking hyperion (2.0.0-alpha.11~buster) ...
Setting up hyperion (2.0.0-alpha.11~buster) ...
--- Hyperion ambient light postinstall ---

-----------------------------------------------------------------------------
---> Hyperion has been installed!
--->
---> You can start Hyperion from your menu now
---> For configuration, visit with your browser: 192.168.178.91:8090
---> or if already used by another service try: 192.168.178.91:8091
-----------------------------------------------------------------------------
Webpage: www.hyperion-project.org
Forum: www.hyperion-project.org
Documenation: docs.hyperion-project.org
-----------------------------------------------------------------------------

```

Output from `export`

```
declare -x DBUS_SESSION_BUS_ADDRESS="unix:path=/run/user/1000/bus"
declare -x DISPLAY="localhost:10.0"
declare -x HOME="/home/pi"
declare -x LANG="en_GB.UTF-8"
declare -x LOGNAME="pi"
declare -x LS_COLORS="rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.wim=01;31:*.swm=01;31:*.dwm=01;31:*.esd=01;31:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:"
declare -x MAIL="/var/mail/pi"
declare -x NO_AT_BRIDGE="1"
declare -x OLDPWD
declare -x PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/games:/usr/games"
declare -x PWD="/home/pi"
declare -x SHELL="/bin/bash"
declare -x SHLVL="1"
declare -x SSH_CLIENT="2a04:4540:6c14:d100:19e5:bb46:af7f:3a28 65179 22"
declare -x SSH_CONNECTION="2a04:4540:6c14:d100:19e5:bb46:af7f:3a28 65179 2a04:4540:6c14:d100:14cd:6f70:9c0:8754 22"
declare -x SSH_TTY="/dev/pts/0"
declare -x TERM="xterm"
declare -x TEXTDOMAIN="Linux-PAM"
declare -x USER="pi"
declare -x XDG_RUNTIME_DIR="/run/user/1000"
declare -x XDG_SESSION_CLASS="user"
declare -x XDG_SESSION_ID="1"
declare -x XDG_SESSION_TYPE="tty"

```

Thanks for all your support and great work! I really appreciate it

**Comment by N1c093** [comment]
I found the solution.... The solution is pretty stupid.

I use the tool MobaXterm to connect to my raspberry. By default the feature `X11-forwarding` is turned on. So the gui based content is passed through the ssh connection.

After disabling this feature, the installation works fine!

![image](https://user-images.githubusercontent.com/59510296/136275357-2f48303f-1b38-405b-ac04-de26e1832d5a.png)

Maybe this also helps somebody else.

**Comment by Paulchen-Panther** [comment]
@mrdarrengriffin 
Can this issue be closed or is there still a problem with the service?

**Comment by Firemanjoe** [comment]
I've also updated to Alpha 11 (both as an upgrade and as a fresh Hyperion install) and the hyperiond.service is missing and I'm not sure how to enable it or to provide root permission as per this error: 

`Device disabled, device 'ws281x' signals error: 'Hyperion must run with "root" privileges for this device. Current user is: "pi"'`

**Comment by Lord-Grey** [comment]
@Firemanjoe 

> I'm not sure how to enable it or to provide root permission as per this error:

You might have seen in the releasenotes that we are providing a script to change the user to root now.

**Comment by Xitee1** [comment]
I have the same issue after upgrading from 2.0.0-alpha.11 to 2.0.12

**Comment by D34DC3N73R** [comment]
The service isn't created for me either on alpha.11 or 2.0.12. I'm running a pi4 2GB headless. It was a manual install on alpha 9. added the apt repository and installed alpha 10, apt upgrade to alpha 11, apt upgrade to 2.0.12.
```
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Raspbian
Description:	Raspbian GNU/Linux 10 (buster)
Release:	10
Codename:	buster
```

I've tried the remove, purge, update, install but that doesn't help either. I've also tried removing the `.hyperion` config folder in my home directory, but that doesn't help either. I've also tried disabling x11 forwarding to no avail.
